src/components/Graph/interfaces.tsx

```diff
@@ -0,0 +1,64 @@
+export interface Data {
```

Comment:
This file has wrong extension: change it to `.ts`
tsx is just for components, we will add this to contribution markdown